### ISSUE TITLE
Download GRCh38 databases for version 3.2.6

### ISSUE THREAD

**Issue by geocarvalho** [Illumina/Nirvana#68]
Hello team, thank you for this nice project.
I'm using version 3.2.6 and I would like to download the databases for GRCh38 reference.
```
$ dotnet bin/Release/net6.0/Nirvana.dll --version
Nirvana 3.2.6 Cache version: 1, Supplementary annotation version: 50, Reference version: 7
```

I couldn't find the `Download.dll` file to try the command recommended in the documentation:
```
$ dotnet bin/Release/net-version/Downloader.dll --ga GRCh38  -o Data
```

There is any way I can download the data using the new release?

Best,
George.

**Comment by geocarvalho** [comment]
Hello, just to update that I'm using `v.3.17.0` now and I could download the GRCh38 databases.

**Comment by rajatshuvro** [comment]
Hi @geocarvalho , 
Sorry for the late reply. The Downloader is not available for 3.2.6 unfortunately. 

Thanks
Rajat

**Comment by ynedelec** [comment]
Hey guys,

@rajatshuvro could you please put a note somewhere in the documentation ?
Had the same issue as @geocarvalho and thought I was dumb. 😉
